benchmark-ph1ex1/3D-model/postprocessing.py

```diff
@@ -0,0 +1,236 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import os
+import matplotlib.patches as mpatches
+from matplotlib.cbook import get_sample_data
+from matplotlib.patches import RegularPolygon
+from matplotlib.collections import PatchCollection
+from matplotlib.pyplot import gca
+from matplotlib.axes import Axes
+import matplotlib.ticker as ticker
+
+
+def geo_label():
```

Comment:
This function is very large and hardcodes many variables. That's fine, but please don't copy paste this into other folders in order to use it on other output files. Instead, modify it to make it more generic so that you DRY.

Reply:
I'll give it a try!